### ISSUE TITLE
Document current handling of deletes

### ISSUE THREAD

**Issue by corylown** [pod4lib/aggregator#439]
Question from @JohnMarkOckerbloom :

> The documentation at https://github.com/pod4lib/aggregator/wiki/Data-requirements#record-delete-data notes that deletes can either be specified with a text file of identifiers (the case I knew about) or by a MARC file that has delete codes in the leader of its records.  In the latter case, an uploaded MARC file may include new and updated records, or deleted records, and a processor would need to examine the file before determining which.  Is it permissible for an uploaded MARC file to include both?  Or for an uploaded compound file (such as a .tar.gz file) to include both files with new/updated MARC records and files with MARC deletes?

**Comment by corylown** [comment]
Initial notes:  it’s possible to combine adds/updates/deletes in one file of marc records and any records with d in position 5 of the leader will be marked as deleted. We might want to discourage people from combining multiple files in a single archive (tar file), because I think the processing order is hard to control.

**Comment by JohnMarkOckerbloom** [comment]
Some documentation on deletion behavior is in the "Streams and their files" document at https://github.com/pod4lib/aggregator/wiki/Streams-and-their-files/ (though it doesn't currently explicitly say that deletes and new/changed records can be included in the same tile, or that it's discouraged).  It may be sufficient just to update this file with whatever further information on deletions seems worth specifying or documenting.

**Comment by corylown** [comment]
@JohnMarkOckerbloom I've spent some time exercising the delete functionality in POD Aggregator. We've also fixed a couple of delete related bugs (https://github.com/pod4lib/aggregator/pull/471 and https://github.com/pod4lib/aggregator/pull/462).

It is safe to supply deletes in the the `text/plain` format as a list of ids, as a file of MARC delete records (using the `d` leader code in position 5), or as a mix of MARC record adds, updates, and deletes in a single file as long as the leader code indicates the delete. I'm going to update the docs to reflect these findings and close this issue.
